app/server.go

```diff
@@ -84,28 +84,6 @@ func (cw *CorsWrapper) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 const TIME_TO_WAIT_FOR_CONNECTIONS_TO_CLOSE_ON_SERVER_SHUTDOWN = time.Second
 
-type VaryBy struct {
-	useIP   bool
-	useAuth bool
-}
-
-func (m *VaryBy) Key(r *http.Request) string {
-	key := ""
-
-	if m.useAuth {
-		token, tokenLocation := ParseAuthTokenFromRequest(r)
-		if tokenLocation != TokenLocationNotFound {
-			key += token
-		} else if m.useIP { // If we don't find an authentication token and IP based is enabled, fall back to IP
-			key += utils.GetIpAddress(r)
-		}
-	} else if m.useIP { // Only if Auth based is not enabed do we use a plain IP based
-		key = utils.GetIpAddress(r)
-	}
-
-	return key
-}
```

Comment:
Functionality moved elsewhere in https://github.com/mattermost/mattermost-server/pull/8152